.cargo/config

```diff
@@ -1,2 +1,9 @@
+[patch.'https://github.com/matrix-org/matrix-rust-sdk']
+matrix-sdk-base = { path = "../matrix-rust-sdk/crates/matrix-sdk-base" }
+matrix-sdk-common = { path = "../matrix-rust-sdk/crates/matrix-sdk-common" }
+matrix-sdk-crypto = { path = "../matrix-rust-sdk/crates/matrix-sdk-crypto" }
+matrix-sdk-indexeddb = { path = "../matrix-rust-sdk/crates/matrix-sdk-indexeddb" }
+matrix-sdk-qrcode = { path = "../matrix-rust-sdk/crates/matrix-sdk-qrcode" }
```

Comment:
This might not build for everyone (including CI)?

Reply:
oh sorry, that wasn't meant to be here.